src/Bridge/Symfony/Bundle/EventListener/EventDispatcher.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Symfony\Bundle\EventListener;
+
+use Symfony\Component\EventDispatcher\Event;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+
+/**
+ * @author Alan Poulain <contact@alanpoulain.eu>
+ */
+final class EventDispatcher
+{
+    private $eventName;
+    private $eventClass;
+    private $dispatcher;
+
+    public function __construct(string $eventName, string $eventClass, EventDispatcherInterface $dispatcher)
```

Comment:
so this dispatcher can handle only _one_ event ? you would have for X events (`read`, `write`, ... and so on) X declared services ? Wouldn't that be a mess ? :O